app/config/config-default.js

```diff
@@ -32,23 +32,23 @@ module.exports = {
     // border color (window, tabs)
     borderColor: '#333',
 
-    // custom css to embed in the main window
+    // custom CSS to embed in the main window
     css: '',
 
-    // custom css to embed in the terminal window
+    // custom CSS to embed in the terminal window
     termCSS: '',
 
-    // set to `true` (without backticks and without quotes) if you're using a
```

Comment:
Please keep this. We had so much user errors like:
`showHamburgerMenu: 'true',`

Reply:
Done and added to a few more places as well so it is consistent.